dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/log/LogClient.java

```diff
@@ -179,27 +178,14 @@ public byte[] getLogBytes(String host, int port, String path) {
      * @param path path
      * @return remove task status
      */
-    public Boolean removeTaskLog(@NonNull Host host, String path) {
+    public void removeTaskLog(@NonNull Host host, String path) {
         logger.info("Remove task log from host: {} logPath {}", host, path);
         RemoveTaskLogRequestCommand request = new RemoveTaskLogRequestCommand(path);
         try {
             Command command = request.convert2Command();
-            Command response = this.client.sendSync(host, command, LOG_REQUEST_TIMEOUT);
-            if (response != null) {
-                RemoveTaskLogResponseCommand taskLogResponse =
-                        JSONUtils.parseObject(response.getBody(), RemoveTaskLogResponseCommand.class);
-                return taskLogResponse.getStatus();
-            }
-            return false;
-        } catch (InterruptedException ex) {
-            Thread.currentThread().interrupt();
-            logger.error(
-                    "Remove task log from host: {}, logPath: {} error, the current thread has been interrupted",
-                    host, path, ex);
-            return false;
+            client.send(host, command);
```

Comment:
Because we change it from sync to async, can we add a callback and log the error if there is any? Otherwise it's hard to debug if it failed to remove the task log.

Reply:
Good succession, I change this to `sendAsync` and add callback to log the result.